src/app/Scenes/MyAccount/MyAccountEditEmail.tsx

```diff
@@ -33,6 +36,13 @@ const MyAccountEditEmail: React.FC<{ me: MyAccountEditEmail_me; relay: RelayProp
       onSave={async (dismiss) => {
         try {
           await updateMyUserProfile({ email })
+
+          if (email !== me.email) {
+            toast.show("Please confirm your new email for this update to take effect", "bottom", {
```

Comment:
the double toast is a known issue that will be fixed with the new app shell, might suggest using a centered toast for the time being
```suggestion
            toast.show("Please confirm your new email for this update to take effect", "center", {
```
```suggestion
            toast.show("Please confirm your new email for this update to take effect", "bottom", {
```